### DISCUSSION TITLE
Powering the board ?

### DISCUSSION THREAD

**Discussion by thiensu1122** [HefnySco/OBAL#7] [Q&A] [answered]
Hi,
May I ask how to power the board? I think in the current version I need to connect 5V and ground from Power Delivery Board. Then it poses another issue with the powering system (one extra module), it would be nice if you can include the Power Delivery Board into the current board. There would be some problems with the noise I think. 

P/s: I love your idea and execution, please keep up the good work.

**Comment by HefnySco** [comment]
Hi, Thank you for this post.

I power RPI using two method.
for Rover abd similar vehicles such as boats I use a Power Bank connected to the USB power socket of RPI.
for Drone I use HK BEC 5V and connect it directly to 5V & GND Pins of RPI. It works well withh all RPIs from zero to 4.
![141](https://user-images.githubusercontent.com/4184373/149774743-d2f4cdc2-1aa7-4788-9f20-25b5eae02288.jpg)

**Reply by HefnySco** [reply]
The reason I hesitate to add it as part of the board is that this is a DIY board / dev board and people will have hard time soldering SMD components needed for such a device.

**Comment by bao-olli** [comment, marked as answer]
Thank you @HefnySco, I got it.
